src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -194,6 +194,17 @@ static Library()
                     returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
                     targetFunction: NumberToBoolean)
             },
+            {
+                BuiltinFunctionsCore.BooleanB,
+                StandardErrorHandling<StringValue>(
```

Comment:
>StringValue [](http://example.com/codeflow?start=38&length=11)

BooleanValue

Reply:
Since we don't have Implementation of TextToBoolean with BooleanValue, sticking with StringValue as this is actually not used but rather a placeholder.